plugins/defaultplugins/srplugin/README.md

```diff
@@ -0,0 +1,47 @@
+# SR plugin
+
+The `srplugin` is a Core Agent Plugin designed to configure Segment routing for IPv6 (SRv6) in the VPP.
+Configuration managed by this plugin is modelled by [srv6 proto file](../common/model/srv6/srv6.proto).
+
+All configuration must be stored in ETCD using the srv6 key prefix:
+
+```
+/vnf-agent/<agent-label>/vpp/config/v1/srv6
+```
+
+**Configuring Local SIDs**\
```

Comment:
You should use *second*/*third*/.. (`##/###/...`) level of headers instead of bold text, so it can be linked directly and better formatted.

Reply:
fixed in 6d849269192d77356a0f8d15cfd262679dc5a26e